src/configuration-client/src/test/java/ee/ria/xroad/common/conf/globalconf/ConfigurationDownloaderTest.java

```diff
@@ -51,6 +56,8 @@ public class ConfigurationDownloaderTest {
     private static final int MAX_ATTEMPTS = 5;
     private static final String LOCATION_URL_SUCCESS = "http://www.example.com";
     private static final String LOCATION_HTTPS_URL_SUCCESS = "https://www.example.com";
+    private static final String HOSTNAME_VERIFICATION_ENABLED = "xroad.configuration-client.global_conf_hostname_verification";
```

Comment:
`SystemProperties.CONFIGURATION_CLIENT_GLOBAL_CONF_HOSTNAME_VERIFICATION`could be reused instead of this declaration